Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1152,83 +996,93 @@ private async Task<PipelineOutcome> RunStagedHoudini(Program program, PipelineSt
+          }
+        }
+        else
+        {
+          printer.WriteErrorInformation(errorInfo, tw);
+        }
+      }
+    }
+
+    internal static ErrorInformation GetOutcomeError(ExecutionEngineOptions options, ConditionGeneration.Outcome outcome, string implName, IToken implTok, string msgIfVerifies,
+      TextWriter tw, uint timeLimit, List<Counterexample> errors)
     {
       ErrorInformation errorInfo = null;
 
-      switch (outcome)
-      {
+      switch (outcome) {
         case VCGen.Outcome.Correct:
-          if (msgIfVerifies != null)
-          {
+          if (msgIfVerifies != null) {
             tw.WriteLine(msgIfVerifies);
           }
+
           break;
         case VCGen.Outcome.ReachedBound:
-          tw.WriteLine($"Stratified Inlining: Reached recursion bound of {Options.RecursionBound}");
+          tw.WriteLine($"Stratified Inlining: Reached recursion bound of {options.RecursionBound}");
           break;
         case VCGen.Outcome.Errors:
         case VCGen.Outcome.TimedOut:
-          if (implName != null && implTok != null)
-          {
+          if (implName != null && implTok != null) {
             if (outcome == ConditionGeneration.Outcome.TimedOut ||
-                (errors != null && errors.Any(e => e.IsAuxiliaryCexForDiagnosingTimeouts)))
-            {
-              errorInfo = ExecutionEngine.ErrorInformationFactory.CreateErrorInformation(implTok,
-                string.Format("Verification of '{1}' timed out after {0} seconds", timeLimit, implName), requestId);
+                (errors != null && errors.Any(e => e.IsAuxiliaryCexForDiagnosingTimeouts))) {
+              string msg = string.Format("Verification of '{1}' timed out after {0} seconds", timeLimit, implName);
+              errorInfo = ErrorInformation.Create(implTok, msg, null);
             }
 
             //  Report timed out assertions as auxiliary info.
-            if (errors != null)
-            {
+            if (errors != null) {
               var cmpr = new CounterexampleComparer();
               var timedOutAssertions = errors.Where(e => e.IsAuxiliaryCexForDiagnosingTimeouts).Distinct(cmpr).ToList();
               timedOutAssertions.Sort(cmpr);
-              if (0 < timedOutAssertions.Count)
-              {
+              if (0 < timedOutAssertions.Count) {
                 errorInfo.Msg += $" with {timedOutAssertions.Count} check(s) that timed out individually";
               }
 
-              foreach (Counterexample error in timedOutAssertions)
-              {
+              foreach (Counterexample error in timedOutAssertions) {
                 var callError = error as CallCounterexample;
                 var returnError = error as ReturnCounterexample;
                 var assertError = error as AssertCounterexample;
                 IToken tok = null;
                 string msg = null;
-                if (callError != null)
-                {
+                if (callError != null) {
```

Comment:
Why reformat these?

Reply:
I think the IDE did this when I extracted the code into a separate method